dbgpt/storage/knowledge_graph/community_summary.py

```diff
@@ -348,8 +371,13 @@ async def asimilar_search_with_scores(
 
             if document_graph_enabled:
                 keywords_for_document_graph = keywords
-                for vertex in subgraph.vertices():
-                    keywords_for_document_graph.append(vertex.name)
+                if similar_search_enabled:
+                    for vertex in subgraph.vertices():
+                        vector = await self._garph_embedder.embed(vertex.name)
```

Comment:
In the document graph for tracing the source chunks, only entity names can be used, and the embeddings of entity names cannot be used.